### ISSUE TITLE
"NoConfigurationException" in ajax form

### ISSUE THREAD

**Issue by Klemart3D** [PrestaShop/PrestaShop#16021]
**Describe the bug**
I have a "NoConfigurationException" in back-office product page with PS 1.7.6.1 when I'm trying to add an associated product. It's a 500 error related to rooter. I think I have this issue because I'm using nginx as web server and I have a mistake in my configuration but I can't find where.

```
Symfony\Component\Routing\Exception\ NoConfigurationException
in var/cache/dev/appDevDebugProjectContainerUrlMatcher.php (line 4854) 
```
**To Reproduce**
1. With an nginx server and mode debug to ON, go to a product page in BO
2. Add an associated product, by typing some letters in form
3. See a Symfony red error in profiler, deploid and click on status 500 link 
4. See error "NoConfigurationException" (cf second screenshot)

**Screenshots**
![error-500-product-page](https://user-images.githubusercontent.com/7419525/67097817-357be080-f1bb-11e9-9c19-cf06c6225dd7.png)

![no-configuration-exception](https://user-images.githubusercontent.com/7419525/67097865-4fb5be80-f1bb-11e9-9d8e-c3c4ee459d67.png)

**Additional information**
PrestaShop version: 1.7.6.1
PHP version: 7.2
Web server: Nginx 1.15.9 

May be it can related to #15949 because of "appDevDebugProjectContainerUrlMatcher" found in logs.

**Comment by khouloudbelguith** [comment]
Hi @Klemart3D,

This issue occurs due to a custom module `configurator`.
Thanks to check the issue with the module author & feedback.

Thanks!

**Comment by Klemart3D** [comment]
This module is developed by DM Concept. May be it's not very well compatible with last PS version.
I contacted the developer team, I will let you inform about their response...

**Comment by khouloudbelguith** [comment]
@Klemart3D, thanks!
Waiting for your feedback.

**Comment by khouloudbelguith** [comment]
Hi @Klemart3D,

Despite our several trials, we could not reproduce your issue with the provided information.
It seems that your issue is not a PrestaShop's core bug but most likely a server configuration or customization problem.

Thanks!

**Comment by Klemart3D** [comment]
Hi, I received a call today from a developer of Configurator module but for him error 500 is not linked to the module but maybe to the nginx config or to some PrestaShop's issues that must be fixed before like #15333 or #15329. So it's not fixed at this day, but we will try to apply patches or wait for PS 1.7.6.2...

**Comment by arirangz** [comment]
@Klemart3D Hello !
I have the same issue.
Have you solved it? I'm on 1.7.6.5

Thanks
